.eslintrc.yaml

```diff
@@ -10,14 +10,13 @@ overrides:
   - files: "**/__tests__/**"
     rules:
       no-magic-numbers: off
-      # There are several errors falling under these rules; resolve
       "@typescript-eslint/require-await": warn
       "@typescript-eslint/no-misused-promises": warn
-      "@typescript-eslint/no-unsafe-return": warn
+      "@typescript-eslint/no-unsafe-return": off
       "@typescript-eslint/restrict-plus-operands": warn
       "@typescript-eslint/restrict-template-expressions": warn
-      "@typescript-eslint/explicit-function-return-type": warn
-      "@typescript-eslint/unbound-method": warn
+      "@typescript-eslint/explicit-function-return-type": off
+      "@typescript-eslint/unbound-method": off
```

Comment:
I'm not sure that we should disable these lint rules: 
- `@typescript-eslint/no-unsafe-return`
- `@typescript-eslint/explicit-function-return-type`
- `@typescript-eslint/unbound-method`

These issues usually appear when functions are not returning the right type (in the case of `no-unsafe-return`) or if they don't add an explicit return type. Keeping these rules enabled will help contributors avoid anti-patterns in new code.

Reply:
These were turned off specifically for tests, see above those lines how they affect only test files. This was mostly done because a lot of our tests have a lot of `undefined` and `null` being passed in which causes a bunch of misleading errors on our test files which makes things harder to debug when there are broken tests.

Reply:
Ah I see, sorry about that - I missed the files property at the top. Thanks for clarifying!